src/views/TradingCompetition/components/StyledLaurel.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
+import { Laurel } from '../svgs'
```

Comment:
The Laurel has been added to the UI kit - it'll be imported from there eventually, just have it here for working with.